code/core/system/proxy-velodyne64/include/ProxyVelodyne64.h

```diff
@@ -22,53 +22,50 @@
 
 #include <memory>
 
-#include <opendavinci/odcore/base/module/TimeTriggeredConferenceClientModule.h>
-#include "velodyne64Decoder.h"
 #include "opendavinci/odcore/wrapper/SharedMemory.h"
-#include <opendavinci/odcore/io/udp/UDPReceiver.h>
+#include "velodyne64Decoder.h"
+#include <opendavinci/odcore/base/module/TimeTriggeredConferenceClientModule.h>
 #include <opendavinci/odcore/io/udp/UDPFactory.h>
+#include <opendavinci/odcore/io/udp/UDPReceiver.h>
 
 namespace opendlv {
 namespace core {
 namespace system {
 namespace proxy {
 
-    using namespace std;
-    using namespace odcore::wrapper;
+using namespace std;
+using namespace odcore::wrapper;
+
+class ProxyVelodyne64 : public odcore::base::module::TimeTriggeredConferenceClientModule {
+   private:
+    ProxyVelodyne64(const ProxyVelodyne64 & /*obj*/) = delete;
+    ProxyVelodyne64 &operator=(const ProxyVelodyne64 & /*obj*/) = delete;
+
+   public:
+    /**
+     * Constructor.
+     *
+     * @param argc Number of command line arguments.
+     * @param argv Command line arguments.
+     */
+    ProxyVelodyne64(const int &argc, char **argv);
 
-    class ProxyVelodyne64 : public odcore::base::module::TimeTriggeredConferenceClientModule {
-       private:
-        ProxyVelodyne64(const ProxyVelodyne64 & /*obj*/) = delete;
-        ProxyVelodyne64 &operator=(const ProxyVelodyne64 & /*obj*/) = delete;
+    virtual ~ProxyVelodyne64();
 
-       public:
-        /**
-         * Constructor.
-         *
-         * @param argc Number of command line arguments.
-         * @param argv Command line arguments.
-         */
-        ProxyVelodyne64(const int &argc, char **argv);
+    odcore::data::dmcp::ModuleExitCodeMessage::ModuleExitCode body();
 
-        virtual ~ProxyVelodyne64();
-        
-        odcore::data::dmcp::ModuleExitCodeMessage::ModuleExitCode body();
+   private:
+    void setUp();
+    void tearDown();
+    string memoryName;    //Name of the shared memory
```

Comment:
Please add a new line and private: here as well to separate methods from fields.